apps/e2e/cypress/e2e/genericTemplates.cy.ts

```diff
@@ -727,6 +727,128 @@ context('GenericTemplates tests', () => {
       cy.contains(genericTemplateTitle);
     });
 
+    it('User should be able to copy previous genericTemplate and proceed if isCompleteOnCopy is true', () => {
+      cy.updateCall({
+        id: initialDBData.call.id,
+        ...updatedCall,
+        templateId: createdTemplateId,
+        proposalWorkflowId: workflowId,
+      });
+      cy.createProposal({ callId: initialDBData.call.id }).then((result) => {
+        if (result.createProposal) {
+          cy.updateProposal({
+            proposalPk: result.createProposal.primaryKey,
+            title: proposalTitle[0],
+            abstract: faker.lorem.words(3),
+            proposerId: initialDBData.users.user1.id,
+          });
+          cy.createGenericTemplate({
+            proposalPk: result.createProposal.primaryKey,
+            title: genericTemplateTitle,
+            templateId: createdGenericTemplateId,
+            questionId: createdQuestion1Id,
+          });
+        }
+      });
+
+      cy.login('user1', initialDBData.roles.user);
+      cy.visit('/');
+
+      cy.contains('New proposal', { matchCase: false }).click();
+      cy.get('[data-cy=call-list]').find('li:first-child').click();
+
+      cy.get('[data-cy=title] input').type(proposalTitle[1]);
+
+      cy.get('[data-cy=abstract] textarea').first().type(faker.lorem.words(2));
+
+      cy.contains('Save and continue').click();
+
+      cy.finishedLoading();
+
+      cy.contains(copyButtonLabel).click();
+
+      cy.contains(copyButtonLabel);
```

Comment:
Is this a duplicate, as I dont understand the use of having cy.contains(copyButtonLabel); again?